crates/rune/src/compile/unit_builder.rs

```diff
@@ -495,10 +495,14 @@ impl UnitBuilder {
                     ConstValue::String(meta.item.item.to_string()),
                 );
             }
+
             MetaKind::Function { .. } => (),
             MetaKind::Closure { .. } => (),
             MetaKind::AsyncBlock { .. } => (),
-            MetaKind::Const { .. } => (),
+            MetaKind::Const { const_value } => {
+                self.constants
+                    .insert(Hash::constant(&meta.item.item.to_string()), const_value.clone());
```

Comment:
Hm. I think this needs to use a type path so that nested constants are supported?

Reply:
Yepp! It's a bit weird to use `type_hash` here which feels weird, but yes. Maybe it should be `path`?

Reply:
Since we are renaming things - go for it! It might even make sense to rename the `Item` type into `Path` even if it might be minorly misconstrued as a filesystem path.

Reply:
QualifiedName? :-)

Reply:
(but renaming anything goes into another PR -- I'll keep type_hash for now and remove Hash::constant.)